fmpsdk/alternative_data.py

```diff
@@ -1,49 +1,48 @@
 import typing
+import os
 
 from .url_methods import __return_json_v4
 
+# Read API key from environment variable
+API_KEY = os.environ.get('FMP_API_KEY')
 
-def commitment_of_traders_report_list(
-    apikey: str,
-) -> typing.Optional[typing.List[typing.Dict]]:
+def commitment_of_traders_report_list(apikey: str = None) -> typing.Optional[typing.List[typing.Dict]]:
     """
     Query FMP /commitment_of_traders_report/list API.
 
     List of symbols for COT.
-
-    :param apikey: Your API key.
+    :param apikey: Your API key. If not provided, the function will use the API_KEY from environment variables.
     :return: A list of dictionaries.
     """
     path = f"commitment_of_traders_report/list"
-    query_vars = {"apikey": apikey}
+    query_vars = {"apikey": apikey} if apikey else {"apikey": API_KEY}
```

Comment:
```suggestion
    query_vars = {"apikey": apikey or API_KEY}
```
nit
should work and be a little more concise but doesn't really matter

Reply:
also wondering (maybe not in the scope of this work) if an error should be raised if neither an `apikey` nor `API_KEY` are supplied